### ISSUE TITLE
Can't Specify Github Enterprise URL

### ISSUE THREAD

**Issue by gnboorse** [lvarayut/github-file-icons#1]
I love the extension, and it's great, but it's restricted to Github.com only. I want to be able to have icons on my [internal Github Enterprise domain name].com as well.

Is there an easy way to do this?

**Comment by lvarayut** [comment]
I'm glad that it's helpful to you 😄 . To be honest, I've never used Github Enterprise before and don't have any access to test it. The easiest way would be to add an `option.html` page that allows a user to enter a Github Enterprise domain and we programmatically request a permission for the given domain, something similar to this [PR](https://github.com/sindresorhus/refined-github/pull/450/files#diff-d8c9b164ba48e0b42dc88f735bdfc13aR15). PR is very welcome!

**Comment by lvarayut** [comment]
Github Enterprise is now supported in https://github.com/lvarayut/github-file-icons/commit/acf5ed49b9c8b08df7dfa6501b8a2233e32c415b, released in `v1.1.2`. Just access your Github Enterprise page, right click on the extension icon, and choose Enable Github Enterprise

**Comment by hipstersmoothie** [comment]
in the above referenced commit why didn't you just do:

```
 	"content_scripts": [{
 		"matches": [
			"http://github.com/*",
			"https://github.com/*",
			"https://*.github.com/*",
 		],
```
